qiita_db/test/test_study.py

```diff
@@ -585,9 +586,6 @@ def test_ebi_submission_status_setter(self):
             new.ebi_submission_status = "unknown"
 
     def test_retrieve_info(self):
```

Comment:
Do we really need this test? Basically, is doing the same that the one above but instead of checking that the data that was just inserted is there, you are testing against the one that the populate sql has ....